omero/developers/PythonBlitzGateway.rst

```diff
@@ -80,6 +80,40 @@ loading).
        survivin
     >>> conn.close()
 
+
+Unicode
+"""""""
+
+When unwrapping omero.model rstring attributes as shown above,
+the BlitzObjectWrapper subclasses will decode the utf8-encoded
+string and return the resulting unicode string.
+
+However, some methods added to BlitzObjectWrappers
+will return strings as follows::
```

Comment:
string(s)

Reply:
methods -> strings. Both are plural. This doesn't imply that a single method returns multiple strings.